apps/client/src/hoc/withAddress.js

```diff
@@ -1,18 +1,18 @@
 import React, { useContext } from "react";
 import { Redirect } from "react-router-dom";
-import Context from "../context";
+import { SessionContext } from "../context";
 import { routes, geturl } from "../routes";
 import withTopic from "./withTopic";
 
 const withAddress = (Component) =>
   withTopic(({ ...rest }) => {
-    const { address } = useContext(Context);
+    const { address } = useContext(SessionContext);
     const { topic } = rest;
     if (!address) {
```

Comment:
Maybe we should check for the topic slug in address here.

Reply:
Fixed